src/components/search/components/searchMenu/SearchBar.tsx

```diff
@@ -36,11 +36,11 @@ const SearchBar = () => {
                 SearchByQuery(form.q);
             }}
         >
-            <Form className={styles.searchBarBlock}>
+            <Form className="pb-3">
                 <Row>
                     <Col className="pe-2">
                         <Field type="text" name="q"
-                            className={`${styles.searchBar} w-100 px-2 py-1`}
+                            className={`${styles.searchBar} rounded-c w-100 px-2 py-1`}
```

Comment:
Tom can you explain why in most cases you got rid of the styles object but on other cases we use both the styles object and additional classes.

Reply:
Yes I can! So we still keep some of the custom classes in there as they are necessary, because some CSS attributes are so underused or specific it is not useful to create a custom generic class for it. So, sometimes an element can have both generic class names as a custom one that adds a little bit of specialty.

Another use case is when we use an active class. The active class is used to indicate some element is in an 'active' state and the CSS should change to indicate this activeness. Whilst the class itself is not in the CSS file anymore, the active class is and we need to define the class name in order to get the active effect. That is why we add the custom class on the element, and then only add the active class's effect in the CSS file, like so:

```
.searchBar.active {
    backgroundColor: red;
}
```

Reply:
Ok clear, thanks!